tests/app/admin_Token_GET_test.py

```diff
@@ -52,6 +52,7 @@ def test_normal_1(self, client: TestClient, session: Session):
             "owner_address": "0x56f63dc2351BeC560a429f0C646d64Ca718e11D6",
         }
         self.insert_listing_data(session, token)
+        session.commit()
```

Comment:
For the tests under `tests/app/`, it was necessary to clearly separate the DB session used in test cases from the one used in the API. Therefore, I made modifications to commit the DB session used in test cases only after adding data.